internal/datastore/post_test.go

```diff
@@ -29,19 +30,21 @@ func TestGetPostsByDiscussionID(t *testing.T) {
 	//	mockDatastore := &delphisDB{
```

Comment:
testing testing testing. was battling sqlmock on prepared statements. will dive in deeper once I confirm that this is the path we are taking